hammr/utils/hammr_utils.py

```diff
@@ -240,3 +240,31 @@ def create_user_ssh_key(api, login, sshKey):
         printer.out("Impossible to create sshKey ["+mySshKey.name+"]", printer.ERROR)
         return 2
     return key
+
+def is_uri_based_on_appliance(uri):
+    match = re.match( r'users/.+/appliances/*', uri)
```

Comment:
It will work but this regex is the same as 'users/.+/appliances', the "/*" is useless and you probably mean "/?".
You can eventually used something like:
users/[^/]+/appliances/[0-9]+($|/)
which mean it either terminate without a / for uri like users/guest/appliances/12, or it contains a / after the appliance id (and potentially other character after).